docs/ECLLanguageReference/ECLR_mods/Templ-OPTION.xml

```diff
@@ -488,8 +488,95 @@
               'nan' (only currently supported for real numbers) creates a
               quiet NaN, which will propagate through any real expressions it
               is used in. You can use NOT ISVALID(x) to test if the value is a
-              NaN. Integer and decimal division by zero continue to return 0.
-              </entry>
+              NaN. Integer and decimal division by zero continue to return
+              0.</entry>
+            </row>
+
+            <row>
+              <entry><emphasis>outputLimitMb</emphasis></entry>
+
+              <entry>Default 10 [MB]</entry>
+
+              <entry>Limit of output to a workunit in MB.</entry>
+            </row>
+
+            <row>
+              <entry><emphasis>maxCsvRowSizeMb</emphasis></entry>
+
+              <entry>Default 10 [MB]</entry>
+
+              <entry>Upper limit of a CSV lin e read in MB.</entry>
+            </row>
+
+            <row>
+              <entry><emphasis>compressInternalSpills</emphasis></entry>
+
+              <entry>Default true</entry>
+
+              <entry>Compress internal spills. (e.g., spills created by
+              lookahead or sort gathering). </entry>
+            </row>
+
+            <row>
+              <entry><emphasis>hdCompressorType</emphasis></entry>
+
+              <entry>Default 'FLZ'</entry>
+
+              <entry>Distribute compressor to use.</entry>
+            </row>
+
+            <row>
+              <entry><emphasis>hdCompressorOptions</emphasis></entry>
+
+              <entry>Default ''</entry>
+
+              <entry>Distribute compressor options (e.g., AES key)</entry>
+            </row>
+
+            <row>
+              <entry><emphasis>splitterSpill</emphasis></entry>
+
+              <entry>Default -1</entry>
+
+              <entry>Integer value to indicate whether to force splitters to
+              spill or not. [1 = force spill | 0 = force in memory | -1 =
+              adhere to helper setting ]</entry>
+            </row>
+
+            <row>
+              <entry><emphasis>loopMaxEmpty</emphasis></entry>
+
+              <entry>Default 1000</entry>
+
+              <entry>Max # of iterations that LOOP can cycle through without
+              results before reporting an error</entry>
+            </row>
+
+            <row>
+              <entry><emphasis>smallSortThreshold</emphasis></entry>
+
+              <entry></entry>
```

Comment:
Should mention default..
Which is 0, meaning disabled.

(Separate issue, but should probably be a non-zero default)